internal/crdao/services/event_processing/process_event_stream.go

```diff
@@ -58,11 +59,24 @@ func (c *ProcessEventStream) Execute(ctx context.Context) error {
 		return err
 	}
 
+	syncDaoSetting := settings.NewSyncDAOSettings()
+	syncDaoSetting.SetCasperClient(c.GetCasperClient())
+	syncDaoSetting.SetVariableRepositoryContractStorageUref(c.GetDAOContractsMetadata().VariableRepositoryContractStorageUref)
+	syncDaoSetting.SetEntityManager(c.GetEntityManager())
+
+	for _, setting := range settings.DaoSettings {
+		syncDaoSetting.SetSetting(setting)
+		if err := syncDaoSetting.Execute(); err != nil {
+			zap.S().With(zap.String("setting", setting)).Info("failed to sync DAO setting")
+		}
+	}
```

Comment:
Maybe let's encapsulate this logic and have:
- `SyncDAOSetting` for single value
- `SencDAOSettings` for all values to init the state